docs/guide/cli.md

```diff
@@ -90,7 +90,11 @@ Certain files might need to be changed during the preparation of a build
 job. In that situation, `lage` can ignore those files when calculating what
 has changed with the `--since` flag.
 
-  
+#### include-dependencies
+_type: boolean_
+
+Include all transitive dependencies when running a command regardless of --scope, or --since.
```

Comment:
This may be understood in this way: when this option is on, the --scope and --since options are ignored. Is this what we mean?

Reply:
Yes, it's vulnerable to misinterpretation. I yanked it from the [lerna filter options](https://www.npmjs.com/package/@lerna/filter-options#--include-dependencies) docs.
The explanation after with the examples are quite useful so I will do the same here.